library/src/device/generator/generator.kernel.hpp

```diff
@@ -1158,7 +1158,7 @@ namespace StockhamGenerator
             // Function signature
             if(NeedsLargeTwiddles())
             {
-                str += "template <typename T, StrideBin sb, bool TwdLarge, CallbackType cbtype, "
+                str += "template <typename T, StrideBin sb, CallbackType cbtype, bool TwdLarge, "
```

Comment:
This was switched for some reason?

Reply:
I put cbtype before TwdLarge when callbacks were merged.  But that was after sbcc was moved to the new generator, so the old generator wasn't updated.  This change is updating the old generator to match what the new generator is now doing.